docs/high/dataset.rst

```diff
@@ -214,6 +214,11 @@ axes using ``None``::
 
     >>> dset = f.create_dataset("unlimited", (10, 10), maxshape=(None, 10))
 
+While integers can be passed to ``maxshape`` for one-dimensional datasets, a tuple
+of length 1 must be passed for an unlimited maximum shape of the 1D-shaped dataset,
+i.e. ``(None,)``. Using only ``None`` instead of an integer will lead to the default
```

Comment:
instead of a **tuple**

Reply:
> instead of a **tuple**


Makes sense, but I rather meant when you want to turn a limited maximum shape, given by an integer, into an unlimited maximum shape. In this case you have to replace the integer with a tuple, not with ``None``. Probably I have rephrase it to make it clear.

Reply:
(disclaimer: I'm not a maintainer, so mine is just a user-opinion ;-)
May I propose:

Please note that for unlimited one-dimensional datasets the `(None, )` tuple cannot be substituted by a bare `None` because this would be interpreted as the default value of `maxshape`, i.e. having a `maxshape` identical to `shape`.

Feel free to improve my English, which I know is pretty poor.

Reply:
I like this one! I will edit the docs accordingly!

Reply:
Sorry, it took me almost forever. It just fell back in priorities of my tasks. I'm happy that I could keep track of it. (sorry for the poor commit message. I was just too quick)